plugin/instance/plugin.go

```diff
@@ -248,6 +252,31 @@ func (p *plugin) listDroplets() ([]godo.Droplet, error) {
 	return droplets, nil
 }
 
+const cloudInitTemplate = `
+#cloud-config
+{{ $config := dict "runcmd" .lines }}
+{{ $config | yamlEncode }}
+`
+
+func buildCloudInit(args ...string) (string, error) {
+	t, err := template.NewTemplate("str://"+cloudInitTemplate, template.Options{})
+	if err != nil {
+		return "", err
+	}
+	lines := []string{}
+	for _, l := range args {
+		// split the line
+		for _, ll := range strings.Split(l, ";") {
+			t := strings.Trim(ll, " \t\n")
+			if strings.Index(t, "#!") != 0 {
+				// exclude shebangs like #!/bin/bash
```

Comment:
Curious; shebangs are not supported? (was wondering if here's cases where it makes a difference, e.g. where bash is needed, not plain `/bin/sh`